src/components/with-assessments/redux/actions.ts

```diff
@@ -0,0 +1,94 @@
+import {
+  GET_ASSESSMENTS_REQUESTED,
+  GET_ASSESSMENTS_SUCCEEDED,
+  LOAD_MORE_ASSESSMENTS_REQUESTED,
+  LOAD_MORE_ASSESSMENTS_SUCCEEDED,
+  LOAD_MORE_ASSESSMENTS_FAILED,
+  GET_ASSESSMENTS_FAILED
+} from './action-types';
+
+import type { PagedAssessments } from '../../../types';
+
+export function getPagedAssessmentsRequested(
+  catalogId: string,
+  entityType: string,
+  context: string,
+  page: number
+) {
+  return {
+    type: GET_ASSESSMENTS_REQUESTED,
+    payload: {
+      catalogId,
+      entityType,
+      context,
+      page
+    }
+  };
+}
+
+export function getPagedAssessmentsSucceeded(
+  pagedAssessments: PagedAssessments
+) {
+  return {
+    type: GET_ASSESSMENTS_SUCCEEDED,
+    payload: {
+      assessments: pagedAssessments.content,
+      totalAssessments: pagedAssessments.totalElements,
+      assessmentPageSize: pagedAssessments.size,
+      assessmentsPage: pagedAssessments.number,
+      hasMoreAssessments:
+        pagedAssessments.number + 1 < pagedAssessments.totalPages
+    }
```

Comment:
Det kan være like greit å videresende page og hente ut data i reducer, siden action creator skal være dummest mulig

```suggestion
    payload: {
      assessments: pagedAssessments.content,
      totalAssessments: pagedAssessments.totalElements,
      assessmentPageSize: pagedAssessments.size,
      assessmentsPage: pagedAssessments.number,
      hasMoreAssessments:
        pagedAssessments.number + 1 < pagedAssessments.totalPages
    }
```

Reply:
f1bfe51